examples/source/user-consent/Geolocation-based_Consent_Flow.html

```diff
@@ -75,51 +75,98 @@
 </head>
 <body>
   <!-- ## Setting up amp-geo -->
-  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the present `eea` which includes all countries of the European Economic Area. -->
+  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the iso code `us` which represents the United States, `preset-eea`, and [`unknown`](https://amp.dev/documentation/components/amp-geo/?format=websites#operation)-->
   <amp-geo layout="nodisplay">
     <script type="application/json">
       {
         "ISOCountryGroups": {
-          "eu": [ "preset-eea" ]
+          "eea": [ "preset-eea", "unknown" ],
+          "us": [ "us" ]
         }
       }
     </script>
   </amp-geo>
 
+  
   <!-- ## Defining the consent flow -->
   <!--
-    We define a basic consent flow, similar to [this sample](/content/amp-dev/documentation/examples/documentation/Basic_User_Consent_Flow.html). he flow should only trigger for users in the EU, so we set the flag: `"promptIfUnknownForGeoGroup": "eu"`.
+    The flow should rely on amp-geo to determine which config to show based upon the user's location, so we set the flag: [`geoOverride`](/content/amp-dev/documentation/components/reference/amp-consent.md).
+
+    If amp-geo finds that the user is in the US country group, then the contents of the US config will override the top-level consent config (and same for EEA).
+
+    Our final config for an EEA (and unknown) user would look like this: 
+    ```json
+    {
+      "consentInstanceId": "world-wide-consent",
+      "consentRequired": true,
+      "promptUI": "eea-consent-ui",
+      "postPromptUI": "post-consent-ui"
+    }
+    ```
+    Since `consentRequired` is `true` and `promptUI` is configured, `amp-consentwill show the prompt if no localstorage decision is found.
+
+    Our final config for a US user would look like this: 
+    ```json
+    {
+      "consentInstanceId": "world-wide-consent",
+      "consentRequired": "remote",
+      "checkConsentHref": "/documentation/examples/api/get-consent-sever-side",
+      "promptUI": null,
+      "postPromptUI": "post-consent-ui"
+    }
+    ```
+    If no localstorage decision is found and since `consentRequired` is 'remote', `amp-consent` will wait until it gets a response from `checkConsentHref`. In this example, no matter what the response returned, it will not show anything because `promptUI` was overriden to `null` (since there could be an external site to manage user consent).
+
+    And our final config for a user in neither of those geo groups would be: 
+    ```json
+    {
+      "consentInstanceId": "world-wide-consent",
```

Comment:
Worth mentioning that `consentInstanceId` can not be overridden. But up to you since we've mentioned it in the amp-consent.md